src/nvim/version.h

```diff
@@ -1,6 +1,23 @@
 #ifndef NVIM_VERSION_H
 #define NVIM_VERSION_H
 
+// defined in version.c
+extern char* Version;
+extern char* mediumVersion;
+extern char* longVersion;
```

Comment:
It's not your fault, and I don't expect you to fix this, but it make me sad to see "Version" followed by "lowerMixedCase" variable name.  It also makes me wonder if we should try and prefix global variables in some way, but that's a topic for another day.